src/graphql/pull_request.rs

```diff
@@ -3,36 +3,232 @@ use std::fmt;
 use anyhow::Context as AnyhowContext;
 use async_graphql::{
     connection::{query, Connection, EmptyFields},
-    Context, Object, Result, SimpleObject,
+    scalar, Context, Object, Result, SimpleObject, ID,
 };
+use jiff::Timestamp;
+
+use crate::graphql::DateTimeUtc;
+
+scalar!(PullRequestState);
+scalar!(PullRequestReviewState);
 
 use crate::database::{self, Database, TryFromKeyValue};
+use crate::github::{
+    GitHubCommentConnection, GitHubCommitConnection, GitHubLabelConnection, GitHubPullRequests,
+    GitHubReviewConnection, GitHubReviewRequestConnection, GitHubUserConnection,
+    PullRequestReviewState, PullRequestState, RepositoryNode, ReviewRequestNode,
+};
+
+#[derive(SimpleObject, Debug)]
+pub(crate) struct User {
+    pub(crate) login: String,
+}
+
+#[derive(SimpleObject)]
+pub(crate) struct Label {
+    pub(crate) name: String,
+}
+
+#[derive(SimpleObject, Debug)]
+pub(crate) struct Comment {
```

Comment:
The structures used in `Comment` and `CommentNode` are clearly different (`GitHubPullRequests` vs. `PullRequest`), yet their naming is quite similar, which might confuse users. Could you please rename them to ensure a clear distinction?